code/game/objects/items/weapons/nt_melee.dm

```diff
@@ -13,20 +14,43 @@
 	price_tag = 300
 	matter = list(MATERIAL_BIOMATTER = 25, MATERIAL_STEEL = 5)
 
+/obj/item/weapon/tool/sword/nt/equipped(mob/living/M)
+	. = ..()
+	if(is_held() && is_neotheology_disciple(M))
+		embed_mult = 0.1
+	else
+		embed_mult = initial(embed_mult)
+
+/obj/item/weapon/tool/sword/nt/shortsword
+	name = "NT Gladius"
+	desc = "A saint looking sword, made to do God's work."
+	icon = 'icons/obj/nt_melee.dmi'
+	icon_state = "nt_shortsword"
+	item_state = "nt_shortsword"
+	force = WEAPON_FORCE_DANGEROUS * 1.25
```

Comment:
I don't like modifying force defines like this, i would put the flat damage value and `(WEAPON_FORCE_DANGEROUS * 1.25)` as a comment beside it.

Reply:
https://github.com/discordia-space/CEV-Eris/pull/5918#issuecomment-777182806